deepmd/entrypoints/model_devi.py

```diff
@@ -0,0 +1,156 @@
+from typing import Tuple
+import numpy as np
+from deepmd import DeepPotential
+from deepmd.utils.data import DeepmdData
+        
+
+def calc_model_devi_f(fs):
+    '''
+        fs : numpy.ndarray, size of `n_models x n_frames x n_atoms x 3`
+    '''
+    fs_mean = np.mean(fs, axis=0)
+    fs_err = np.sum((fs - fs_mean) ** 2, axis=-1)
+    fs_devi = np.mean(fs_err, axis=0) ** 0.5
+    max_devi_f = np.max(fs_devi, axis=1)
+    min_devi_f = np.min(fs_devi, axis=1)
+    avg_devi_f = np.mean(fs_devi, axis=1)
+    return max_devi_f, min_devi_f, avg_devi_f
```

Comment:
Isn't it the std for force **components**? But in the C++ API and LAMMPS, the std for **atomic** forces is calculated.

Reply:
Exactly. Thanks for pointing it out.

Reply:
It is the std of atomic forces. I will use `np.linalg.norm` and `np.std` function to make it more clear.